gen3-integration-tests/services/indexd.py

```diff
@@ -78,7 +78,13 @@ def delete_record_via_api(
                 "Authorization": f"bearer {access_token}",
             }
         else:
-            headers = pytest.auth_headers[user]
+            auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
```

Comment:
can this also be restructured like in line 56 above?

Reply:
Done